aspnetcore/blazor/components/render-modes.md

```diff
@@ -105,110 +105,61 @@ To apply a render mode to a component instance use the [`@rendermode` Razor dire
 
 ## Apply a render mode to a component definition
 
 To specify the render mode for a component as part of its definition, use the [`@rendermode` Razor directive](xref:mvc/views/razor#rendermode) and the corresponding render mode attribute.
 
 ```razor
 @page "..."
-@rendermode RenderMode.InteractiveServer
+@rendermode InteractiveServer
 ```
 
-<!-- UPDATE 8.0 Remove NOTE at RTM -->
-
-> [!NOTE]
-> In an app generated from the Blazor project template during *Release Candidate 2*, add a `using static` statement for <xref:Microsoft.AspNetCore.Components.Web.RenderMode> to `Components/_Imports.razor` if you want to use shortened syntax (`@rendermode InteractiveServer`):
->
-> ```razor
-> @using static Microsoft.AspNetCore.Components.Web.RenderMode
-> ```
-
 Applying a render mode to a component definition is commonly used when applying a render mode to a specific page. Routable pages by default use the same render mode as the router component that rendered the page.
 
 Technically, `@rendermode` is both a Razor *directive* and a Razor *directive attribute*. The semantics are similar, but there are differences. The `@rendermode` directive is on the component definition, so the referenced render mode instance must be static. The `@rendermode` directive attribute can take any render mode instance.
 
 > [!NOTE]
 > Component authors should avoid coupling a component's implementation to a specific render mode. Instead, component authors should typically design components to support any render mode or hosting model. A component's implementation should avoid assumptions on where it's running (server or client) and should degrade gracefully when rendered statically. Specifying the render mode in the component definition may be needed if the component isn't instantiated directly (such as with a routable page component) or to specify a render mode for all component instances.
 
-> [!NOTE]
-> During .NET 8 *Release Candidate 2*, the Blazor Web App project template sets the interactive render mode with the [`@attribute` Razor directive](xref:mvc/views/razor#attribute) in template-generated sample components. The template will be updated to use the [`@rendermode` Razor directive](xref:mvc/views/razor#rendermode) for the release of .NET 8 in November.
->
-> Render mode             | Directive
-> ----------------------- | ---------
-> Interactive Server      | `@attribute [RenderModeInteractiveServer]`
-> Interactive WebAssembly | `@attribute [RenderModeInteractiveWebAssembly]`
-> Interactive Auto        | `@attribute [RenderModeInteractiveAuto]`
->
-> You can change the directives in an app generated by the project template to use the `@rendermode` directive.
-
 ## Prerendering
 
 Prerendering is enabled by default for interactive components.
```

Comment:
Do we have a writeup somewhere else, explaining what `prerendering` is? I think it's important to have these notes (even sometimes repeating it) so that customers can understand the basic concepts.
Something like the following:

```text
Prerendering is the process of rendering a component or a page on the server, without wiring up any event handlers for any controls being rendered. This is done, so that the server can output the raw HTML UI of the page as soon as possible in response to the initial request.
```

Reply:
Good point because the earliest explanation was in the *Blazor Server* content in the *Overview* that has now been versioned out for this release. For <8.0, it says ...

> After the connection is made between the client and the server, the component's static prerendered elements are replaced with interactive elements. Prerendering the content on the server makes the app feel more responsive on the client.

... and we lost that with the updates that Dan and I made, so I'll re-introduce it to the *Overview*. It was part of the <8.0 article on *Prerendering and integration*, which has now been split. I'll add your remark to the top of the new *Prerender* article at ...

https://learn.microsoft.com/en-us/aspnet/core/blazor/components/prerender?view=aspnetcore-8.0